README.md

```diff
@@ -186,6 +186,23 @@ npm run lint
 npm run format
 ```
 
+### Typescript Definitions (Beta)
+
+Starting with v5.3.0, this project has bundled Typescript definitions included. These definitions are in their infancy and are **not yet recommended for production applications**. We welcome the community's help in maintaining these definitions and contributing improvements as we add missing types or make small corrections.
+
+#### Typescript Exclusions
+
+- We do not provide a DefinitelyTyped version of these definitions at this time
```

Comment:
Nit: add a colon, sub-list the items below

Reply:
The items below are not related to this first bullet point.